### DISCUSSION TITLE
The connection timed out occasionally on the web side in our production environment

### DISCUSSION THREAD

**Discussion by gangchzu** [miguelgrinberg/python-socketio#1220] [Q&A]
**Describe the bug**
We run python-socketio server with tornado，occasionally，the connection timed out on the web side in our production environment，but the test environment cannot reproduce.
On the web side logs show that  it cannot receive the open event of the server after the client is connected, and continues to try to initiate a connection after a 20s timeout， On the server side logs show  that client is connected successfully，could you please give me some ideas?

the connection is based on websocket，not polling

server side logs:
884b05179ecc4592ba2e208020926b66: Sending packet OPEN data {'sid': '884b05179ecc4592ba2e208020926b66', 'upgrades': [], 'pingTimeout': 60000, 'pingInterval': 25000}
884b05179ecc4592ba2e208020926b66 is entering room xxx [/]
884b05179ecc4592ba2e208020926b66: Sending packet MESSAGE data 0
884b05179ecc4592ba2e208020926b66: Received request to upgrade to websocket
884b05179ecc4592ba2e208020926b66: Upgrade to websocket successful

python-socketio version is 4.6.1
python-engineio version is 3.14.2

**Comment by miguelgrinberg** [comment]
Please do not write issues on this project when you just need help debugging your application. The discussions board is the appropriate place for this.

Questions for you:
- What is the version of your client
- Can you share the client logs as well, please. Ideally logs for the same session from the client and server sides.
